pftpd-pojo-lib/build.gradle

```diff
@@ -1,18 +1,18 @@
 apply plugin: 'java'
 
-targetCompatibility = '1.7'
-sourceCompatibility = '1.7'
+targetCompatibility = '1.8'
+sourceCompatibility = '1.8'
 
 repositories {
     jcenter()
 }
 
 dependencies {
-    compile 'org.bouncycastle:bcpg-jdk15on:1.60'
-    compile 'org.bouncycastle:bcpkix-jdk15on:1.60'
+    implementation 'org.bouncycastle:bcpg-jdk15on:1.60'
+    implementation 'org.bouncycastle:bcpkix-jdk15on:1.60'
```

Comment:
Building locally on my box fails :cry: 
Compiler error:
```
org/primftpd/services/PubKeyAuthenticator.java:42: error: package org.bouncycastle.jce.provider does not exist
            if (serverKey instanceof org.bouncycastle.jce.provider.JCEECPublicKey)
```
Are `implementation` jars visible to dependent projects?